tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -109,7 +120,8 @@ def run(file_path, class_name):
         }
         return json_schema
 
-
+#Todo: write docstring for extract and rdf_to_json to properly state that the first is the actual extraction of each
```

Comment:
🔧 I think it's worth taking the time to capture that docstring rather than leaving a todo